evennia/server/inputfuncs.py

```diff
@@ -41,6 +42,28 @@
 def _NA(o):
     return "N/A"
 
+def _strip_unsafe(session, txt):
+    """
+    Removes 'unsafe' codes that might be used to cause other users'
+    terminals to display unintended characters. This strips both
+    ingested HTML (which can affect webclient) as well as nasty
+    text-formatting characters (newline, tab). Server developers
+    can overload this function to add or remove processing steps.
+
+    Args:
+        txt (str) : The incoming input string
+    Returns:
+        txt (str) : A cleansed input string
+    """
+    if session.account:
+        if session.account.permissions.get("Developer"):
+            return txt
+
+    # Strip unsafe characters from ingest
+    txt = strip_tags(txt)  # Any HTML codes
+    txt = re.sub(r'|\/', '', txt)  # New lines
+    txt = re.sub(r'|-', '', txt)  # Tabs
```

Comment:
This is highly specific to Evennia's default ANSI formatter and people can replace it at will (there is even a contrib to do it). Hard-coding these sequences here is not useful. 

Besides, these formatting instructions are expected by users - a user entering a description string in any more RP-heavy game will expect the ability to at least be able to divvy up their text into paragraphs with line break. If we remove them here, what's the point of having these formatting operations at all (if you produce this in code you can just use \n \t directly)?